flowcell_parser/classes.py

```diff
@@ -42,64 +42,64 @@ def parse(self, demultiplexingDir='Demultiplexing'):
             fc_name = m.group(4)
         # For MiSeq we parse the samplesheet "run_folder/SampleSheet_copy.csv"
         if "M0" in instrument:
-            ss_path = os.path.join(self.path, 'SampleSheet_copy.csv')
+            samplesheet_path = os.path.join(self.path, 'SampleSheet_copy.csv')
         else:
-            ss_path = os.path.join(self.path, 'SampleSheet.csv')
-        rinfo_path = os.path.join(self.path, 'RunInfo.xml')
-        rpar_path = os.path.join(self.path, 'runParameters.xml')
+            samplesheet_path = os.path.join(self.path, 'SampleSheet.csv')
+        run_info_path = os.path.join(self.path, 'RunInfo.xml')
+        run_parameters_path = os.path.join(self.path, 'runParameters.xml')
```

Comment:
This file is called "RunParameters.xml" in the NSX I believe.

Reply:
Yes, but TACA creates a link named `runParameters.xml` that points to the one with capital R.